pom.xml

```diff
@@ -521,7 +509,6 @@
     <module>compensations</module>
     <module>osgi</module>
     <module>narayana-full</module>
-    <module>code-coverage</module>
```

Comment:
updated plugins don't allow us to exclude this module but it can't run in main build anyway as it relies on the rts etc coverage to be ran separate